textos/articulo-de-opinion.md

```diff
@@ -0,0 +1,17 @@
+# plus ultra 
+Más allá de la convivencia entre personas está el **respeto**, que podría definirse como pararse a escuchar lo que opina, siente y piensa la persona de al lado. España es un lugar situado en un mundo donde poco a poco la cinta que cubre la boca de miles de personas para no decir lo que piensan está desapareciendo. En España hay un lugar llamado **Euskadi**, un lugar que podría decirse que se encuentra dividido en **dos bandos**.
+
+Este último año con el tema del **independentismo** cantalán ha resurgido la parte más independentista vasca, como si un foco de luz enfocase dejando en penumbra a la otra parte en contra de la independencia. He aquí la cuestión:
+
+* Hace aproximadamente 40 años Euskadi y España despertaban de una larga siesta en la que la libertad de expresión estaba censurada por el gobierno de Franco. No se podía mencionar nada que se opusiera al régimen franquista. Toda la población debía mantenerse en silencio para no ser llevados a los edificios donde muchos estaban por hablar alto y claro. Hoy en día la libertad de expresión es indomable, todos podemos hablar libremente, perfecto, pero ¿dónde dice que tener libertad de expresión es sinónimo de ignorancia, agresión y desconsideración hacia tus paisano?
```

Comment:
Esto que habéis formateado como un elemento de lista, no tiene mucho sentido si no hay más elementos en esta lista. Si lo que queréis es destcarlo, ponedlo todo en negrita, o buscad otra forma de hacerlo.